app/rest/platformroutes.ts

```diff
@@ -180,4 +180,54 @@ export async function platformroutes(
 			});
 		});
 	});
+
+	/**
+	 * *
+	 * Block by block number
+	 * GET /fetchDataByBlockNo
+	 * curl -i 'http://<host>:<port>/fetchDataByBlockNo/<channel_genesis_hash>/<blockNo>'
+	 */
+	router.get('/fetchDataByBlockNo/:channel_genesis_hash/:blockNo', (req, res) => {
+		const blockNo = req.params.blockNo;
```

Comment:
converting blockNo to numeric type.

Reply:
@adityajoshi12 Will change the code as suggested